### DISCUSSION TITLE
Introspect readonly?

### DISCUSSION THREAD

**Discussion by kfrp** [geldata/gel#2146] [Q&A] [answered]
I've read the schema docs but don't find any approach to introspect whether a property or link is readonly.  If it's possible, where would I put it in the following snippet?

```
with module schema select ObjectType { 
        name, 
        properties: { 
          name, 
          cardinality, 
          required,          
          type := .target.name,
          unique := ('std::exclusive' IN .constraints.name),
          default
        }
} FILTER .name LIKE 'mymodule::%';
```
Thanks!

**Comment by elprans** [comment, marked as answer]
Looks like we overlooked this.  Should be easy to fix by adding `CREATE PROPERTY readonly -> std::bool;` to this declaration: https://github.com/edgedb/edgedb/blob/8e5d87e196b9c927a0ea0e4ef29c999d33b6dd44/edb/lib/schema.edgeql#L218-L227
